openid_connect_configuration.conf

```diff
@@ -28,6 +28,18 @@ map $host $oidc_jwt_keyfile {
     default "http://127.0.0.1:8080/auth/realms/master/protocol/openid-connect/certs";
 }
 
+map $host $oidc_end_session_endpoint {
+    default "http://127.0.0.1:8080/auth/realms/master/protocol/openid-connect/logout";
+}
+
+map $host $oidc_logout_extra_query_params {
+    # Each IdP may use different query params of the $oidc_end_session_endpoint.
+    # For example, Amazon Cognito requires `logout_uri=xxx`, and Auth0 requires
+    # `returnTo=xxx` instead of $oidc_logout_callback_param.
+    default "$oidc_logout_callback_param&id_token_hint=$session_jwt";;
```

Comment:
The proposed version did not add simplicity and clarity, but rather the opposite. It's better to return as it was.

Reply:
Yeah. Returned as it was!